src/labels_by_user_input.py

```diff
@@ -32,3 +42,41 @@ def labels_by_user_input(data, pull):
     if f"/un{LABEL_APPROVE}".lower() in body:
         remove_label(pull=pull, label=LABEL_APPROVE)
         set_commit_status_pending_no_approve(commit=last_commit)
+
+    check_for_semver_labels(pull=pull, last_commit=last_commit, body=body)
+
+
+def check_for_semver_labels(pull, last_commit, body):
```

Comment:
`check_for_semver_labels` can you please move it to utils and leave here only the action (set labels/remove and so on)

Reply:
Please see https://github.com/myakove/pr-workflows/commit/b596358df8dd0599173f2a4b741eb85290ceb0bf